spec/requests/v7/assessments_controller_spec.rb

```diff
@@ -0,0 +1,2079 @@
+require "rails_helper"
+
+module V7
+  RSpec.describe AssessmentsController, :calls_bank_holiday, type: :request do
+    describe "POST /create" do
```

Comment:
This massive copy and paste feels wrong

Reply:
@starswan Agree 😄 but i think v7 should include v6 + v7 specific additional tests. Do you think we only include  schema validation related stuff in v7 specs?

Reply:
duplicate v6 specs in v7 has been removed